scalableminds/webknossos
# Fix 3D View initialization

The main problem here was that the input handlers for the 3D are only initialized once and only when starting a tracing in plane mode. 

### Mailable description of changes (needs to be understandable by webknossos mailing list people):
- fixes issues with 3D view

### Steps to test:
1. Start a tracing
2. Zoom, rotate, move in 3D View & click the "XY"... buttons
3. Switch to flight mode and reload page to start the tracing directly in flight mode
4. Observe that stuff still works

### Issues:
- fixes #1851

------
